### DISCUSSION TITLE
Reporting minor issues - like text in description boxes

### DISCUSSION THREAD

**Discussion by pgale** [Clooos/Bubble-Card#447] [General]
Do you want us to report minor formatting and descriptive text issues in the issues section? or somewhere else so as not to clutter up the issues?

For instance, in LIGHT mode only in a browser, the first para of the comment renders as the following with the link text invisible. I scratched my head with that until I realised. Dark mode shows it OK.

![Screenshot 2024-04-14 113820](https://github.com/Clooos/Bubble-Card/assets/513044/f70f6375-94c5-4f9b-afa9-74ffa39d439f)

**Comment by Clooos** [comment]
This is an issue to me, thanks again!
